Automattic/wp-calypso
# Update the Pressable page to disallow the users from selecting a plan if ownership is 'regular.'

Currently, the Pressable plan selection page does not account for the type of ownership. This PR ensures that agency users who have an account linked to an existing Pressable subscription will not be able to select and purchase a plan.

<img width="1728" alt="Screenshot 2024-08-06 at 4 42 19 PM" src="https://github.com/user-attachments/assets/80cc845c-fdf2-4e91-9448-6a2fe2ee7b23">


Closes https://github.com/Automattic/jetpack-genesis/issues/461

## Proposed Changes

* Update the Pressable plan selection component to accommodate the ownership type. If the type is 'regular', then we show a different CTA button that redirects to the Pressable dashboard. We also hide the slider to prevent the user from changing the current plan.

## Testing Instructions

* To easily test this, you can spin up this branch locally.
* Update `pressableOwnership` variable to equal `regular` in [this line](https://github.com/Automattic/wp-calypso/pull/93135/files#diff-ec2f39e59c24f0c351f5e7dd957f7f7bcc694886e56fede10e86112f2a03bac4R59).
* Go to http://agencies.localhost:3000/marketplace/hosting?hosting=premier and confirm the correct CTA button.

## Pre-merge Checklist
